x-pack/legacy/plugins/alerting/server/routes/create.ts

```diff
@@ -16,7 +15,12 @@ interface ScheduleRequest extends Hapi.Request {
     tags: string[];
     alertTypeId: string;
     interval: string;
-    actions: AlertAction[];
+    actions: Array<{
+      group: string;
+      id: string;
+      actionTypeId: string;
```

Comment:
As @pmuellr mentioned, APIs (create / update) shouldn't accept `actionTypeId` and we should add it ourselves while also validating the action.

Reply:
@pmuellr I made the changes for this here: https://github.com/elastic/kibana/pull/51628/commits/005d3d9933a3e4f26e5d7222bb175c290875628c.